config.js

```diff
@@ -35,5 +35,5 @@
     "locale": "en",
     "integrationName": "frontend",
     "useHostAPILogin": true,
-    "version": "2.8.0"
+    "version": "2.9.0"
```

Comment:
We're adding a feature so I think it should be 2.9.0. I dont feel like this is a bug fix. But I can change it if you feel strongly.